src/python/director/continuouswalkingdemo.py

```diff
@@ -151,7 +152,7 @@ def _setupOnce(self):
         #footContactSubContinuous.setSpeedLimit(60)
 
         lcmUtils.addSubscriber('FOOTSTEP_PLAN_RESPONSE', lcmdrc.footstep_plan_t, self.onFootstepPlanContinuous)# additional git decode stuff removed
-        lcmUtils.addSubscriber('IHMC_FOOTSTEP_STATUS', lcmdrc.footstep_status_t, self.onFootstepStatus)
+        lcmUtils.addSubscriber('IHMC_FOOTSTEP_STATUS', ihmc.footstep_status_t, self.onFootstepStatus)
```

Comment:
Currently, there is no footstep_status_t message in drc. This message was renamed from ipab (https://github.com/openhumanoids/oh-distro/commit/82e2fac78efd31d89d312087f821430b0d607691). Instead of using this message, I can look for another solution.